src_kernels/cuda/grid_update.rs

```diff
@@ -125,25 +126,29 @@ fn update_single_cell(
 
                         if apply_friction {
```

Comment:
This could be a better name
```suggestion
                        if apply_friction_and_cancel_penetration {
```